qutebrowser/browser/network/filescheme.py

```diff
@@ -77,14 +85,14 @@ def dirbrowser_html(path):
     if is_root(path):
         parent = None
     else:
-        parent = os.path.dirname(path)
+        parent = os.path.normpath(os.path.join(path, '..'))
```

Comment:
Use `os.pardir` instead of `'..'` here - even though that's currently `..` everywhere, it makes the intent more clear